osbenchmark/utils/dataset.py

```diff
@@ -24,8 +24,7 @@ class Context(Enum):
     INDEX = 1
     QUERY = 2
     NEIGHBORS = 3
-    MAX_DISTANCE_NEIGHBORS = 4
```

Comment:
I'll rebase against main to pull in these changes on the next revision, didn't catch until I submitted the PR. Will also fixup the commits into a single one in the next revision.